### DISCUSSION TITLE
Issues implementing Boostrap 5 Tooltips & Popovers

### DISCUSSION THREAD

**Discussion by cbrowncode** [holger1411/understrap#1890] [Q&A] [answered]
I'm trying to implement Bootstrap 5 pop-overs & tooltips... but can't seem to get it working.
I've set up a child theme. I tried enqueuing popper.min.js separately ahead of the child-theme.min.js... and then adding a script to the page to initialize the poppers... but nothing seems to work. 

Any suggestions?

**Comment by bacoords** [comment, marked as answer]
Have you been able to try any of the recommendations in the existing issues? Here's one issue on it https://github.com/understrap/understrap-child/issues/328 and I think here's the final answer on how to do it with the current build process: https://github.com/understrap/understrap/issues/1583#issuecomment-1022444176

**Reply by cbrowncode** [reply]
Thank you so much for your help! Indeed, [#1583](https://github.com/understrap/understrap/issues/1583#issuecomment-1022444176) was the trick to getting them to work.
